mod_reforged/hooks/skills/actives/gash_skill.nut

```diff
@@ -35,6 +37,16 @@
 			});
 		}
 
+		if (this.m.BleedStacks != 0)
+		{
+			ret.push({
+				id = 6,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Applies " + ::MSU.Text.colorGreen(this.m.BleedStacks) + " stacks of [Bleeding|Skill+bleeding_effect] when inflicting at least " + ::MSU.Text.colorRed(::Const.Combat.MinDamageToApplyBleeding) + " damage")
```

Comment:
How do you define the coloring for thresholds?
Here you use red for a threshold, which is really neither positive nor negative, it's just not clear whether this number is good or bad for the player. 
If your only concern was to highlight this number then I suggest using using colorNeutral for this.

There is not a clear vanilla convention for these numbers when poison_coat_effect colors is red but fearsome perk description does not use color for these thresholds.

Reply:
Anything other than green is acceptable for this here. I think the red works fine as it shows a limit you must overcome for the perk to trigger.